web/api/py/codechecker_api/setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import setup, find_packages
+from os import path
+
+# io.open is needed for projects that support Python 2.7
+# Python 3 only projects can skip this import
+from io import open
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='codechecker_api',
+    version='6.24.0',
+    author='CodeChecker Team (Ericsson)',
+    description='Python API stub package for the CodeChecker server.',
+    long_description_content_type='text/markdown',
+    long_description=long_description,
+    url='https://github.com/Ericsson/codechecker',
+
+    classifiers=[  # Optional
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 3 - Alpha',
+
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        # These classifiers are *not* checked by 'pip install'. See instead
+        # 'python_requires' below.
+        'Programming Language :: Python :: 2',
+    ],
+
+    keywords='codechecker thrift api library',
```

Comment:
It can be a list of strings:
```suggestion
    keywords=['codechecker', 'thrift', 'api', 'library'],
```